apps/rush-lib/src/cli/taskRunner/ProjectTask.ts

```diff
@@ -243,3 +248,43 @@ function _areShallowEqual(object1: Object, object2: Object, writer: ITaskWriter)
   }
   return true;
 }
+
+/**
+ * When running a command from the "scripts" block in package.json, if the command
+ * contains Unix-style path slashes, the package managers will convert the slashes
+ * to backslashes for the Windows OS.  This is a complicated heuristic.  For example,
+ * we want to convert "node_modules/bin/this && ./scripts/that" to
+ * "node_modules\bin\this && .\scripts\that" but we don't want to convert
+ * the slashes in  "cmd.exe /c blah".  NPM and PNPM use npm-lifecycle for this,
+ * but it unfortunately has a dependency on the entire node-gyp kitchen sink.
+ * Yarn has a simplified implementation in fix-cmd-win-slashes.js, but it's not
+ * exposed as a library.
+ *
+ * Since the "&&" and quoting edge cases are mostly deprecated for Rush projects,
+ * we will just do something very basic for now.  We can improve it later if someone
+ * actually complains.
+ */
+export function convertSlashesForWindows(command: string): string {
+  // Match everything up to the first space, "&" or quote
```

Comment:
>Match everything up to the first space, "&" or quote [](start = 5, length = 52)

*Match everything up to the first space, "&" or quote in the first capture group.